bin/configs/typescript-echo-api.yaml

```diff
@@ -0,0 +1,9 @@
+generatorName: typescript
+outputDir: samples/client/echo_api/typescript/build
+inputSpec: modules/openapi-generator/src/test/resources/3_0/echo_api.yaml
+templateDir: modules/openapi-generator/src/main/resources/typescript
+additionalProperties:
```

Comment:
maybe you need to define platform node here?

Reply:
That's it, I think, thanks!